### PR TITLE
[FIX] mail: activity don't loose text of feedback popover

### PR DESCRIPTION
**Pull request by kebeclibre**
On an activity, click Mark as done, the popover pops over
Now write some text in it, but click somewhere else, like to copy/paste something
The popover looses focus, and your text disappears

After this commit, we store the value of the popover when hiding it,
then, when when showing back the popover, the value is reinjected
OPW 1873999

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by aab-odoo** [comment]
ok but it would be better with a test :-)

**Comment by nim-odoo** [comment]
@aab-odoo test added

File: addons/mail/static/tests/chatter_tests.js [hunk 1]
```diff
@@ -160,6 +161,69 @@ QUnit.test('basic rendering', function (assert) {
...
+    });
+
+        // Assign a temporary value to the feedback
+        $feedbackPopover.find('#activity_feedback').val('John Shaft');
+
+        // Make feedback loose focus
+        //form.$('input.o_field_char[name=foo]').click();
+        $feedbackPopover.focusout();
+
+        hiddenPopoverDeferred.then(function () {
```
Comment (aab-odoo) [inline addons/mail/static/tests/chatter_tests.js]: do you really need the deferreds?

**Comment by kebeclibre** [comment]
@aab-odoo 
Lol, sorry for this, it is a WIP ....

**Comment by aab-odoo** [comment]
@kebeclibre @nim-odoo Nico implicitly asked me to review...

**Comment by nim-odoo** [comment]
@aab-odoo my bad, I didn't know it was a WIP

**Comment by kebeclibre** [comment]
@aab-odoo 
Ready to be reviewed

**Comment by kebeclibre** [comment]
@aab-odoo 
squash and changes made

File: addons/mail/static/src/js/activity.js [hunk 2]
```diff
@@ -224,19 +224,38 @@ var Activity = AbstractActivityField.extend({
             })
             .then(this._reload.bind(this, {activity: true}));
     },
+
+    _rebindPopoverFocusout: function ($popover_el) {
```
Comment (aab-odoo) [inline addons/mail/static/src/js/activity.js]: docstring

Comment (aab-odoo) [reply addons/mail/static/src/js/activity.js]: this function should be in the private section,

File: addons/mail/static/src/js/activity.js [hunk 3]
```diff
@@ -224,19 +224,38 @@ var Activity = AbstractActivityField.extend({
...
+        var self = this;
+        var $popover = $popover_el.data("bs.popover").tip();
+        var activity_id = $popover_el.data('activity-id');
+        $popover.off('focusout');
+        $popover.focusout(function (e) {
+            // outside click of popover hide the popover
```
Comment (aab-odoo) [inline addons/mail/static/src/js/activity.js]: hides

File: addons/mail/static/src/js/activity.js (hunk 2 above)
Comment (aab-odoo) [reply addons/mail/static/src/js/activity.js]: I'd say _bind... not _rebind...

**Pull request merged**